pack-psds.py

```diff
@@ -38,7 +38,7 @@
 
     fgrid = np.arange(f0, fmax, df)
     series = lal.CreateREAL8FrequencySeries(
-        psd_file.name, 0, f0, df, lal.SecondUnit, len(fgrid))
+        (psd_file.name).split(".")[0], 0, f0, df, lal.SecondUnit, len(fgrid))
```

Comment:
Why are the PSD entries renamed?

Reply:
I just remove the extension ".txt", 
the extension generates raises errors 
like this :

  File "/home/weizmann.kiendrebeogo/anaconda3/envs/observing_scenarios/lib/python3.9/site-packages/ligo/lw/ligolw.py", line 472, in enc
    raise ValueError("invalid Name '%s'" % name)
ValueError: invalid Name '****avirgo_O3actual.txt'****

After remove the extension the psds name  become avirgo_O3actual..... without  the extension.
and this work .

Reply:
Thanks. I was able to reproduce locally after updating python-ligo-lw.